src/GeositeFramework/js/Export.js

```diff
@@ -105,14 +112,28 @@ require(['use!Geosite'],
                                 .wrapAll("<div class='print-legend-coll'></div>");
                         });
 
+                    function afterPrintHandler() {
+                        _.delay(previewDeferred.resolve, 250);
+                    }
+
+                    /* 
+                        Chrome's exposed 'window.print' method includes a preview and
+                        blocks, whereas non-Chrome browsers do neither; similarly,
+                        Chrome does not expose an 'onafterprint' event while the
+                        others do.
+                    */
+                    if (!isChrome()) {//ie, ff, safari?
```

Comment:
The line comments here and below could be removed now that the code is more self-explanatory.